docs/sidebar.yml

```diff
@@ -63,8 +65,6 @@ sidebar:
               url: docs/reference/contextual/relationship-implicits.html
         - title: Other New Features
           subsection:
-            - title: Multiversal Equality
-              url: docs/reference/other-new-features/multiversal-equality.html
```

Comment:
I think this will break all links to the Multiversal Equality doc, we should keep the old url with a redirect.